nearlib/test/integration.test.js

```diff
@@ -21,6 +25,73 @@ test('test creating default config', async () => {
     Near.createDefaultConfig();
 });
 
+describe("dev connect", () => {
+    let deps;
+    beforeEach(async () => {
+        const keyStore = new InMemoryKeyStore();
+        const storage = createFakestorage();   
+        deps = {
+            keyStore,
+            storage,
+            createAccount: (async (newAccountName, newAccountPublicKey) => {
+                const createAccountResponse = await account.createAccount(newAccountName, newAccountPublicKey, 1, aliceAccountName);
+                await nearjs.waitForTransactionResult(createAccountResponse);
+            })
+        };
+    });
+
+    test('test dev connect with no account creates a new account', async () => {
```

Comment:
It's still somewhat unclear:
1) What is method under test? I.e. at what step you expect failure. Is it `dev.connect` judging by the fact that invalid stuff setup (e.g. `deps.storage.setItem(storageAccountIdKey, await generateUniqueString("invalid"))`) happens before this call.
2) What are assertions specific to what being tested in this test (vs what always need to be true).

Reply:
Can we address this after today? Ideally we push the actual api changes for weekend.